packages/gatsby-cli/src/index.ts

```diff
@@ -45,7 +46,7 @@ if (semver.prerelease(version)) {
   report.warn(
     report.stripIndent(`
     You are currently using a prerelease version of Node (${version}), which is not supported.
-    You can use this for testing, but we do not recommend it in production.
+    You can use this for testing, but we do not recommend it in production. 
```

Comment:
Intentional?

Reply:
https://github.com/gatsbyjs/gatsby/pull/25158/files#r443140668 prettier did this in original PR and because this reverts everything it restore previous one as well